module2/gift-certificate-system/web/src/main/java/com/epam/esm/config/WebAppInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.config;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRegistration;
+
+public class WebAppInitializer implements WebApplicationInitializer {
+    @Override
+    public void onStartup(ServletContext servletContext) throws ServletException {
+//        AnnotationConfigWebApplicationContext applicationContext = new AnnotationConfigWebApplicationContext();
+//        applicationContext.register(SpringConfig.class);
+//        applicationContext.setServletContext(servletContext);
+//
+//        ServletRegistration.Dynamic dispatcher = servletContext.addServlet("dispatcher", new DispatcherServlet(servletContext));
+//        dispatcher.setLoadOnStartup(1);
+//        dispatcher.addMapping("/");
```

Comment:
What is the goal of having this code commented out?

Reply:
My project didn't run with this configuration until I commented it out. I rewrote the configuration, but I still have questions about this topic.